### PR TITLE
Update powered-by.md

### PR DESCRIPTION
**Pull request by lilien1010**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/iresty/apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

**Comment by moonming** [comment]
thx @lilien1010 
can you add comment in this issue too? https://github.com/iresty/apisix/issues/487

**Comment by moonming** [comment]
and please sign the CLA

File: doc/powered-by.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Users are encouraged to add themselves to this page, [issue](https://github.com/
 1. Tencent Cloud 腾讯云 https://cloud.tencent.com/
 1. zuzuche 租租车 https://www.zuzuche.com/
 1. chaolian 超链云商 https://www.chaolian360.com/
+1. HelloTalk.Inc https://www.HelloTalk.com/
```
Comment (juzhiyuan) [inline doc/powered-by.md]: `HelloTalk Inc` would be better.

**Comment by moonming** [comment]
@lilien1010  Please keep the alphabetical order

**Comment by moonming** [comment]
@lilien1010 any update?

**Comment by membphis** [comment]
@lilien1010 any news?

**Comment by lilien1010** [comment]
@membphis  I create a new PR and  close this one。

**Pull request closed without merging**